Source/Relativity.Transfer.SDK.Samples.Core/UI/ConfigurationScreen.cs

```diff
@@ -55,11 +55,18 @@ public ConfigurationScreen(IPathExtension pathExtension)
 					new SourceAndDestinationConfiguration(source, destination));
 
 			case TransferType.UploadFile:
-				source = AnsiConsole.Ask("Source file", configuration.UploadFile.Source);
+				source = AnsiConsole.Ask("Source file", configuration.UploadFile.Source); 
 				destination = AnsiConsole.Ask("Destination directory", configuration.UploadFile.Destination);
 
 				return Configuration.Configuration.ForUploadFile(common,
 					new SourceAndDestinationConfiguration(source, destination));
+			
+			case TransferType.UploadItems:
+				source = AnsiConsole.Ask("Source file", configuration.UploadFile.Source);
```

Comment:
```suggestion
				source = AnsiConsole.Ask("List of items source file (load file)", configuration.UploadFile.Source);
```

Load file is a domain name so our clients know this name.